data/virt_autotest/host_unattended_installation_files/autoyast_opensuse_kvm_sshd.xml

```diff
@@ -68,6 +68,13 @@
   <networking>
     <keep_install_network t="boolean">true</keep_install_network>
     <backend>network_manager</backend>
+    <interfaces t="list">
+      <interface>
+        <bootproto>dhcp</bootproto>
+        <name>p3p1</name>
+        <startmode>manual</startmode>
```

Comment:
Why manual start, not auto? If manual is selected, which part of code will start it? I didn't find it in the PR.

Reply:
this interface is deactivated deleberately. It brings troubles. but it cannot be disabled entirely because it is needed by future sriov tests. The default interface is 'em1' which is used in our regular test.

Reply:
I see, thanks for explanations. It is helpful to add a comment for this?

Reply:
I am not sure if a comment is welcome in an autoyast profile so I describe it in commit msg and PR description.

Reply:
I just checked the commit msg. It is good enough!  BTW, adding comment in autoyast is welcome too, format is <!--> , but not need to comment here again, since it is well mentioned.